modules/images/webp-uploads/load.php

```diff
@@ -493,6 +495,14 @@ function webp_uploads_img_tag_update_mime_type( $image, $context, $attachment_id
 			continue;
 		}
 
+		// If smaller image file is preferred, and the target mime type is a larger file size, do not update.
+		if (
+			true === $prefer_smaller_image_file &&
```

Comment:
Nit-pick: We don't need to be that specific, any truthy value should be accepted, as there's no harm in doing so.
```suggestion
			$prefer_smaller_image_file &&
```